pnpm-lock.yaml

```diff
@@ -7,42 +7,42 @@ settings:
 dependencies:
   '@pinia/nuxt':
     specifier: ^0.5.1
-    version: 0.5.1(typescript@5.4.5)(vue@3.4.31)
+    version: 0.5.5(typescript@5.6.3)(vue@3.5.12)
```

Comment:
copy the pnpm lock file in your branch from main. There should not be any change to lock file without the change in package.json

Reply:
I had done thst because it was getting convoluted. I will do it again